### ISSUE TITLE
CKV2_AWS_6: false positive when doing terraform plan

### ISSUE THREAD

**Issue by dbbc96** [bridgecrewio/checkov#5028]
**Describe the issue**
doing an terraform plan shows that public access should be off however it is in both terraform and verified in console.

**Examples**
Here is section that calls on our module:
```hcl
module "s3_csv_bucket" {
  source = "../modules/terraform-aws-s3-bucket-0.4"

  enabled         = true
  aws-environment = var.environment
  bucket_name     = "${var.agency}-${var.project}-${var.environment}-bucket"
  acl             = "private"

  ignore_public_acls            = true
  block_public_acls             = true
  block_public_policy           = true
  restrict_public_buckets       = true
}
```

This part below is in the module:
public_access_block_enabled = var.block_public_acls || var.block_public_policy || var.ignore_public_acls || var.restrict_public_buckets

**Additional context**
this had worked previously

**Comment by gruebel** [comment]
hey @dbbc96 thanks for reaching out.

Sadly I can't help you much with the given information, either I would need a TF plan, where the scan fails or the module code to recreate it.

**Comment by dbbc96** [comment]
to give some more context i had found in another ticket i opened with you all today that something changed between version 2.3.214 and 223.  This issue for the s3 bucket passes with 214 but fails with 223.

**Comment by gruebel** [comment]
As said, with no code, we can't help.
